deploy/backplane/mtsre/10-mtsre-admins-project.ClusterRole.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-admins-project
+rules:
+# SRE can get, list, and watch all resources
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
We can't allow this. This would allow for reading of all secrets. Why aren't cluster-reader permissions sufficient?